ingest/src/main/java/com/microsoft/azure/kusto/ingest/SecurityUtils.java

```diff
@@ -14,7 +14,7 @@
 import java.security.spec.PKCS8EncodedKeySpec;
 import java.security.spec.X509EncodedKeySpec;
 
-class SecurityUtils {
+public class SecurityUtils {
     private SecurityUtils() {
         // Hide constructor as this is a Utils class
```

Comment:
Comment is no longer relevant

Reply:
Why?

Reply:
Because you no longer hide it - it's public

Reply:
The class is public, but it's still a Utils class, so we don't want them to create an instance of the class. So the constructor should be private.